### PR TITLE
ST-206 Adding the dsconfig tool to the test environment

### PR DESCRIPTION
**Pull request by kmadisa**
This adds the [dsconfig](https://github.com/MaxIV-KitsControls/lib-maxiv-dsconfig) service to the application, which registers devices to the tango database.

For now, I have created a dummy configuration file to use to register the _PowerSupply_ device server to the database.

JIRA: [ST-206](https://jira.skatelescope.org/browse/ST-206)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -53,11 +53,24 @@ services:
...
+    network_mode: ${NETWORK_MODE}
+    container_name: ${CONTAINER_NAME_PREFIX}dsconfig
+    depends_on:
+      - databaseds
+    environment:
+      - TANGO_HOST=${TANGO_HOST}
+    command: >
+      sh -c "wait-for-it.sh ${TANGO_HOST} --timeout=30 --strict --
+             json2tango -w tango_example/powersupply/config.json &&\
+              sleep 3m"
```
Comment (kmadisa) [inline docker-compose.yml]: @Matteo04052017 I'm still trying to see if there is a way to keep this service running after it has registered the device to the database.

**Comment by Matteo04052017** [comment]
I made the following changes:
- added two more devices, 
- expanded the config.json file, putting it outside the powersupply folder,
- added another container for the dump of the database in json,
- replaced the PROJECT (in the makefile) from "powersupply" to "training".

Please update the helm chart. If you need help of need clarification, let me know.

File: docker-compose.yml (hunk 1 above)
Comment (Matteo04052017) [reply docker-compose.yml]: well, that's another story. I know that at max4 they added a grpc interface for it.

File: config.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
+{
```
Comment (kmadisa) [inline config.json]: @Matteo04052017 This config file, should I make a copy of it and place it in the `charts/templates/data/` directory  or just  move it there?

Comment (Matteo04052017) [reply config.json]: Nice question. For the moment, move it in the charts/tango-example/templates/data/ and adjust the docker-compose file for the dsconfig container. 

We'll talk about it tomorrow with @piersharding

**Pull request merged**